calculation/gmhazard_calc/gmhazard_calc/directivity/scripts/mc_nhyp_runs.py

```diff
@@ -24,35 +27,83 @@ def perform_mp_directivity(
     grid_space,
     nhm_dict,
     output_dir,
```

Comment:
I think it might be a good idea to provide the data types.
Even better with some docstrings, I reckon.

Reply:
Yeah, but not really part of this PR. Also docstring is probably not that important in this case, since its not exactly a frequently used function (i.e. only in this script, and this script will pretty much only be used during the directivity investigation.